frontend/src/metabase/setup/selectors.ts

```diff
@@ -82,6 +82,12 @@ export const getIsHosted = (state: State): boolean => {
 };
 
 export const getTokenFeature = (state: State, feature: TokenFeature) => {
+  // We should never have token features in non-EE builds.
+  // This check prevents cases where the token from env is applied to OSS.
+  if (!isEEBuild()) {
```

Comment:
Fixes a bug we saw where if you put in env vars for EE token in an OSS build, it still returns true if that token contains EE feature even if it's OSS